File: noi.openjudge.cn/ch0105/19.py
# http://noi.openjudge.cn/ch0105/solution/39419260/

n = int(input())
c = 0
for i in range(n):
    x, y, a = map(float, input().split())
    b = (((x * x + y * y) ** 0.5)* 2) / 50 + a * 1.5
    c += b
print(int(c + 0.999999))

File: noi.openjudge.cn/ch0104/19.cpp
// http://noi.openjudge.cn/ch0104/solution/40077221/

#include<iostream>
using namespace std;
int main() {
    int a, b;
    char c;
    cin >> a >> b >> c;
    if (c == '+') {
        cout << a + b << endl;
    } else if (c == '-') {
        cout << a - b << endl;
    } else if (c == '*') {
        cout << a * b << endl;
    } else if (c == '/') {
        if (b == 0) {
            cout << "Divided by zero!" << endl;
        } else {
            cout << a / b << endl;
        }
    } else {
        cout << "Invalid operator!" << endl;
    }
        
}
